galaxy_importer/loaders.py

```diff
@@ -217,12 +275,11 @@ def _get_readme(self):
             os.path.join(self.root, self.rel_path))
         if not readme:
             raise exc.ContentLoadError('No role readme found.')
-        self.readme_file = readme.name
-        self.readme_html = markup_utils.get_html(readme)
+        return readme
 
     def _get_metadata_description(self):
         self.log.info('Getting role description')
-        pass
+        return ''
```

Comment:
Whats this method for? Does schema.Content() for a role require a non None description?

Reply:
this method is for getting the role description and will be in an upcoming pr, the value will be added to the contents json... None may be better when it is empty, I will look at it more closely in that pr.